docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -519,7 +519,7 @@
     }
    ],
    "source": [
-    "display(pd.DataFrame(output_data[\"line\"][\"i_from\"][:,0]))"
+    "display(pd.DataFrame(output_data[ComponentType.line][\"i_from\"][:,0]))"
```

Comment:
Worth discussion, probably as a follow up: this `data.line['i_from']` mixed way is a bit awkward imo

Reply:
@Jerry-Jinfeng-Guo Do you mean that having component types with their own data type whereas their attributes being treated as regular strings is strange?

Reply:
Same as above, this mixed way of doing it: component and attribute level.